app/js/components/selectBox/index.jsx

```diff
@@ -5,12 +5,37 @@ import './styles.scss';
 
 class SelectBox extends React.Component {
   state = {
-    name: this.props.name,
-    items: this.props.items || [],
-    selectedItem: this.props.items[0] || this.props.selectedItem,
+    name: '',
+    items: [],
+    selectedItem: null,
     showItems: false,
   }
 
+  static getDerivedStateFromProps(props, state) {
+    if (props.items.length !== state.items.length) {
```

Comment:
I think it will be nice to destructure props and state

Reply:
I cannot destructure both props and state because they have similar properties, so let say I destructure props
```const { name, items, selectedItem } = props;```
when I try to destructure state, those variable would have already been defined.

Reply:
@Rhotimee ☝️

Reply:
Ok. I see. 
Great job.